content/home/participate.md

```diff
@@ -12,7 +12,7 @@ Sign-up for the news and share your expectations using
 This form collects contact emails and also offers opportunities to share your expectations for the project.
```

Comment:
Should we be more specific about what they are interested in? Using, contributing. 

I already got the question what it means to get listed as an interested party.

Reply:
The form asks about types of contributions, but indeed it worth listing it explicitly

Reply:
Should be addressed now